src/Listeners/RouteMatchedListener.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Bugsnag\BugsnagLaravel\Listeners;
+
+use Illuminate\Routing\Events;
+
+class RouteMatchedListener
+{
+    public function onRouteMatched($event)
+    {
+        app()['bugsnag']->getSessionTracker()->createSession();
```

Comment:
This app function doesn't exist. It only exists from the foundation illuminate package that this code does not depend on.

Reply:
As far as I'm aware this is a global helper function and should be available in any of the files used within the laravel framework.

Reply:
It's defined in the foundation package, which does not exist outside of laravel/framework, so this breaks on code bases that are not laravel framework or lumen framework, but still satisify the dependencies of this package as defined in the composer.json.

Reply:
Its also defined in the lumen package, which would only leave users who use laravel/(something) but not laravel/framework and not lumen. That seems reasonable.